apps/cli/src/adapters/yarn/index.ts

```diff
@@ -0,0 +1,16 @@
+import { execa } from "execa";
+import * as process from "node:process";
+
+import { NodeReader, ScriptSchema } from "../../domain/node.js";
+
+export const makeNodeReader = (): NodeReader => ({
+  getScripts: async (cwd = process.cwd()) => {
+    const { exitCode, stdout } = await execa("yarn", ["run", "--json"], {
+      cwd,
+    });
```

Comment:
Is running this command necessary? I think the scripts can be checked just by looking at the `script` section of `package.json` (without executing it or inferring the package manager)